modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -1059,7 +1091,31 @@ struct InferList: public cv::detail::KernelTag {
             return;
         }
 
-        IE::Blob::Ptr this_blob = extractBlob(*ctx, 1);
+        IE::Blob::Ptr this_blob = nullptr;
+
+        if (ctx->uu.params.rctx != nullptr) {
+            GAPI_Assert(ctx->inShape(1) == cv::GShape::GFRAME &&
+                            "Callback feature is supported for MediaFrame only");
+            cv::util::any any_blob_params = ctx->inFrame(1).blobParams();
+            auto ie_core = cv::gimpl::ie::wrap::getCore();
+
+            std::pair<InferenceEngine::TensorDesc,
+                      InferenceEngine::ParamMap> blob_params;
+            try {
+                blob_params = cv::util::any_cast<std::pair<InferenceEngine::TensorDesc,
+                                                           InferenceEngine::ParamMap>>(any_blob_params);
+            } catch (...) {
+                GAPI_Assert(false && "Incorrect type of blobParams: "
+                                      "expected std::pair<InferenceEngine::TensorDesc,"
+                                                         "InferenceEngine::ParamMap>");
+            }
+
+            this_blob = ctx->uu.params.rctx->CreateBlob(blob_params.first,
+                                                        blob_params.second);
+        } else {
+            this_blob = extractBlob(*ctx, 1);
+        }
```

Comment:
what about other types of infer? e.g. InferROI and InferList2? Also, highly recommend to move all this code into `extractBlob()` and keep these actors unaware of these details.

Reply:
Moved to `extractBlob()`. Technically it can work with all types of infer (I'm not sure though). Only the default `Infer` and `InferList` have been tested locally

Reply:
Can you please add relevant tests to avoid surprises?

Reply:
oh you can't.. at least in public and now, oh.